Octokit.Tests.Integration/Clients/CommitStatusClientTests.cs

```diff
@@ -28,6 +28,27 @@ public class TheGetAllMethod
         }
     }
 
+    public class TheGetCombinedMethod
+    {
+        [IntegrationTest]
+        public async Task CanRetrieveCombinedStatus()
```

Comment:
Just changed the existing test. It didn't really test what we wanted (more or less just copied from the test above :stuck_out_tongue:).